core/rend/gles/gldraw.cpp

```diff
@@ -8,6 +8,9 @@
 #ifdef LIBRETRO
 #include "postprocess.h"
 #include "vmu_xhair.h"
+// Add these global variables for the libretro build
```

Comment:
These are constants and should be marked `constexpr`. And they don't need to be defined globally. Move these to where they are used (`drawVmuTexture()` in the `#ifdef LIBRETRO` section)